### PR TITLE
Update to k8s 1.11 libraries, fix code

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

**Comment by justinsb** [comment]
Blocked on #5817

**Comment by rdrgmnzs** [comment]
Looks like kubernetes/client-go 1.11.3 uses strings.Builder which was only added in Golang 1.10. Should we drop Golang 1.9 from the travis tests? It looks like we'll also be adding Golang 1.11 once #5786 and #5781 lands.

**Comment by rdrgmnzs** [comment]
> Looks like kubernetes/client-go 1.11.3 uses strings.Builder which was only added in Golang 1.10. Should we drop Golang 1.9 from the travis tests? It looks like we'll also be adding Golang 1.11 once #5786 and #5781 lands.

And as I push comment, he pushes a diff to do just that. 😄

File: channels/pkg/api/channel.go [hunk 1]
```diff
@@ -17,15 +17,14 @@ limitations under the License.
...
 
 import (
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type Addons struct {
 	metav1.TypeMeta `json:",inline"`
 
 	// We use v1.ObjectMeta so we don't serialize everything, even though we are using the full api machinery here (yet!)
```
Comment (rdrgmnzs) [inline channels/pkg/api/channel.go]: Comment needs to be updated.

Comment (mikesplain) [reply channels/pkg/api/channel.go]: @rdrgmnzs Ahh yeah good eye, if @justinsb doesn't have time to fix that when he fixes the conflict below, would you mind fixing in a follow up?

**Pull request merged**